app/views/RoomView/index.js

```diff
@@ -815,6 +815,23 @@ class RoomView extends React.Component {
 		}
 	});
 
+	closeBanner = async() => {
+		const { room } = this.state;
+		const { rid } = room;
+		try {
+			const db = database.active;
+			const subCollection = db.collections.get('subscriptions');
+			const newRoom = await subCollection.find(rid);
```

Comment:
You don't need to fetch this from database, `this.state.room` is a database reference, you can use this directly.